src/immutable/components/props-to-js.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import type { Component } from 'react';
+
+import React from 'react';
+import { isImmutable } from '../util/immutable';
+
+type Props = {
+  [key: string]: any
```

Comment:
We should probably use generics here in the future instead of `any`–but not necessary now!